templates/docs/examples/patterns/cta/block/default.html

```diff
@@ -0,0 +1,12 @@
+{% extends "_layouts/examples.html" %}
+{% block title %}CTA Block{% endblock %}
+
+{% block standalone_css %}patterns_cta{% endblock %}
+
+{% block content %}
+<hr>
```

Comment:
Let's make border part of the component, so it doesn't depend on HR element in HTML.